lib/StackAnalysis/StackAnalysis.cpp

```diff
@@ -801,6 +802,173 @@ void commitToModel(GeneratedCodeBasicInfo &GCBI,
   revng_check(TheBinary.verify(true));
 }
 
+static UpcastablePointer<model::Type>
+buildPrototype(GeneratedCodeBasicInfo &GCBI,
```

Comment:
Can you put some comments in the body of this function?
I'll review it later, but I already now I'll ask for this.